src/main/scala/generate/database/PostgresGenerator.scala

```diff
@@ -0,0 +1,30 @@
+package generate.database
+
+import generate.database.ast._
+
+object PostgresGenerator extends DatabaseGenerator {
+
+  private def parseColumn(column: Column): String =
+    column match {
+      case IntColumn(name)        => s"    $name INT"
+      case FloatColumn(name)      => s"    $name REAL"
+      case StringColumn(name)     => s"    $name TEXT"
+      case BoolColumn(name)       => s"    $name BOOLEAN"
+      case DateColumn(name)       => s"    $name DATE"
+      case TimeColumn(name)       => s"    $name TIME"
+      case DateTimeTzColumn(name) => s"    $name TIMESTAMPTZ"
+    }
+
+  override def generate(statement: Statement): String = {
+    val sb = new StringBuilder()
+    statement match {
+      case Create(tableName, columns) =>
+        sb.append(s"CREATE TABLE $tableName (\n")
```

Comment:
Hmm.. the way parens are interpolated here leaves some room for error in the future.
Would it be possible to have a function that wraps the provided string in parens?

```scala
def wrap(value: String): String = { .. }
```

Reply:
Maybe also with newlines, but less worried about that

Reply:
Or just add the brackets together to the mkString